src/server/routes/api/document.js

```diff
@@ -0,0 +1,117 @@
+import Express from 'express';
+import uuid from 'uuid';
+import Cytoscape from 'cytoscape';
+import CytoscapeSyncher from '../../../model/cytoscape-syncher';
+import { BASE_URL } from '../../env';
+import { importCX, exportCX } from '../../../model/import-export/cx';
+import { importJSON, exportJSON } from '../../../model/import-export/json';
+
+const http = Express.Router();
+
+const makeNetworkId = () => 'cy' + uuid();
+
+/**
+ * Post (create) a new network
+ * @param {Function} importBody A function that takes (cy, body) and converts the body to cy
+ * @param {Express.Request} req The HTTP request
+ * @param {Express.Response} res The HTTP response
+ * @param {Express.NextFunction} next The Express next(err) function
+ */
+const postNetwork = async (importBody, req, res, next) => {
+  try {
+    const body = req.body;
+    const id = makeNetworkId();
+    const cy = new Cytoscape();
+
+    cy.data({ id });
+
+    importBody(cy, body);
```

Comment:
```suggestion
    importBody(cy, body);
    
    cy.data({ id });
```

Addresses Mike's comment re. ID.  The server-generated UUID is always used.